npc/jobs/2-1/assassin.txt

```diff
@@ -553,8 +546,9 @@ OnTouch_:
 				mes "Alright then,";
 				mes "best of luck to you!";
 				close2;
-				set ASSIN_Q3,1;
-				set ASSIN_Q,1;
+				if (!ASSIN_Q3)
+					ASSIN_Q3 = 1;
```

Comment:
For me this change is incorrect, ASSIN_Q3 should be assigned regardless of the previous value (same for other change)